CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -16,6 +16,11 @@ public class HarvestGatewayTests
         private const string HarvestAccountId = "123456";
         private const string UserAgent = "The Wolves";
 
+        private static readonly string Roles =
+            "Software Engineer, Senior Software Engineer, Senior Engineer, Lead Engineer, "+
+                "Delivery Manager, SRE, Consultant, Delivery Principal";
+        
```

Comment:
Extra blank line.

Reply:
```suggestion
```

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/31/commits/00f4b59914005bca399412f767614ae41feebd50.
Cheers!